homeassistant/components/knx/schema.py

```diff
@@ -855,6 +859,7 @@ class SensorSchema(KNXPlatformSchema):
             vol.Optional(CONF_STATE_CLASS): STATE_CLASSES_SCHEMA,
```

Comment:
We should move that below entity category 🔡

Reply:
Yeah sure, I just noticed it could be sorted alphabetically for easier reading. If you don't want to merge as it is 🤷